pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[tool.poetry]
+name = "openslide-python"
+version = "1.1.2"
+description = "Python interface to OpenSlide"
+authors = []
+maintainers = ["OpenSlide project <openslide-users@lists.andrew.cmu.edu>"]
+license = "LGPL-2.1-only"
+readme = "README.md"
+homepage = "https://openslide.org/"
+repository = "https://github.com/openslide/openslide-python"
+keywords = [
+    "openslide",
+    "whole-slide",
+    "image",
+    "virtual slide",
+    "library"
+]
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Healthcare Industry',
+    'Intended Audience :: Science/Research',
+    'Operating System :: MacOS :: MacOS X',
+    'Operating System :: Microsoft :: Windows',
+    'Operating System :: POSIX :: Linux',
+    'Topic :: Scientific/Engineering :: Bio-Informatics'
+]
+packages = [
+    { include = "openslide" },
+]
+include = [
+    { path = "*.txt", format = "sdist" },
+    { path = "doc/*.py", format = "sdist" },
```

Comment:
should the sdist tarball contain things like documentation, examples and tests? or should it just be setup.py + the bits needed to compile a module from scratch when there is no wheel?

Reply:
It should at least include anything needed by a downstream package.  I'd expect packages to ship docs and examples (some omit the latter because of the bundled JS), and some do actually run the test suite.